demo/stream/platform/posix/server.c

```diff
@@ -0,0 +1,53 @@
+// Copyright 2020 Hugo Lindström <hugolm84@gmail.com>
+
+// This software is supplied under the terms of the MIT License, a
+// copy of which should be located in the distribution where this
+// file was obtained (LICENSE.txt).  A copy of the license may also be
+// found online at https://opensource.org/licenses/MIT.
```

Comment:
I think you should make the server once, and use the STREAM APIs.  I realize part of what you might want to show is that these work with ordinary TCP/IP applications, but we want to show that the STREAMs API can be used on both the server and the client side.

Reply:
If you don't have time for that, let me know, and I'll work on it.

Reply:
Yes, my point was to show that nng can be used even with legacy applications that uses TCP/IP over a standard OS socket. I did not think of showcasing that nng_stream can be used as a server as well. Perhaps showcasing both have its own value? In any case, I can make some time to implement the server with STREAM.

Reply:
I think showcasing both has value.  Plus that way its just one program not two for the server.  Showing the portability benefit is good. :-)